libs/viewer/src/Automation.cpp

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by mIcable law or agreed to in writing, software
```

Comment:
mIcable -> applicable (I think Automation.h, jsonParseUtils.h and SimpleViewer.h have this mutation too)

Reply:
Wups, will fix!